README.md

```diff
@@ -47,245 +17,134 @@ After you're finished please remove all the comments and instructions!
     - [Key Features](#key-features)
   - [🚀 Live Demo](#live-demo)
 - [💻 Getting Started](#getting-started)
-  - [Setup](#setup)
   - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
   - [Install](#install)
   - [Usage](#usage)
   - [Run tests](#run-tests)
-  - [Deployment](#triangular_flag_on_post-deployment)
+  - [Deployment](#deployment)
 - [👥 Authors](#authors)
 - [🔭 Future Features](#future-features)
 - [🤝 Contributing](#contributing)
 - [⭐️ Show your support](#support)
 - [🙏 Acknowledgements](#acknowledgements)
-- [❓ FAQ](#faq)
+- [❓ FAQ (OPTIONAL)](#faq)
 - [📝 License](#license)
 
-<!-- PROJECT DESCRIPTION -->
+<!-- DESCRIPTION -->
+## 📖 About the Project <a name="#about-project"></a>
 
-# 📖 [your_project_name] <a name="about-project"></a>
+In the upcoming projects, you will use a relational database to create the data structure for a vet clinic. You will start with one table, and step by step, you will create a complete database with data about:
 
-> Describe your project in 1 or 2 sentences.
+    animals
+    animals' owners
+    clinic employees
+    visits
 
-**[your_project__name]** is a...
+Apart from building data structure, you will be challenged with tasks that require SQL queries to answer specific questions.
 
-## 🛠 Built With <a name="built-with"></a>
 
-### Tech Stack <a name="tech-stack"></a>
-
-> Describe the tech stack and include only the relevant sections that apply to your project.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<details>
-  <summary>Client</summary>
-  <ul>
-    <li><a href="https://reactjs.org/">React.js</a></li>
-  </ul>
-</details>
+## 🛠 Built With <a name="built-with"></a>
 
-<details>
-  <summary>Server</summary>
-  <ul>
-    <li><a href="https://expressjs.com/">Express.js</a></li>
-  </ul>
-</details>
+### Tech Stack <a name="tech-stack"></a>
 
 <details>
-<summary>Database</summary>
+  <summary>PostgreSQL</summary>
   <ul>
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+    <li><a href="https://www.postgresql.org">PostgreSQL</a></li>
   </ul>
 </details>
 
-<!-- Features -->
-
-### Key Features <a name="key-features"></a>
+<!-- FEATURES -->
 
-> Describe between 1-3 key features of the application.
+### ⭐️ Key Features <a name="key-features"></a>
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+You will build your database in your local environment, and Microverse will not require you to share it. Therefore only you will have access to its data (which is a good thing from the security point of view). However, you will need to share all SQL queries you use to build and query your database to get feedback about them.
```

Comment:
- [ ] [ ] [OPTIONAL] This description isn't also a valid key feature so I'd suggest you provide the core features that you think are important in this project and list them here.

Reply:
I fixed the key features section.

![03](https://github.com/jlcarrascof/Vet-clinic-database/assets/86327867/bc093b30-f460-4c7d-9817-29052602b61a)